components/post_markdown/system_message_helpers.jsx

```diff
@@ -37,7 +37,7 @@ function renderFormattedText(value, options) {
 }
 
 function renderJoinChannelMessage(post) {
```

Comment:
Just a note for me, I believe these renderers and others that were included in the combined system messages can be safely removed as they are not being used anymore  Will clean it up separately.